python/GangaLHCb/Lib/RTHandlers/XMLWorkerScript.py

```diff
@@ -1,5 +1,8 @@
 # XMLSummary parsing
 import os, sys
+
+from GangaLHCb.Lib.Applications.AppsBaseUtils import activeSummaryItems
```

Comment:
Rats, I should have spotted this sooner. This is not correct, we do NOT ship ganga to the Worker Node.